packages/tinacms/src/components/media/media-manager.tsx

```diff
@@ -80,6 +95,10 @@ export function MediaPicker({
     if (cms.media.isConfigured) return 'loading'
     return 'not-configured'
   })
+  const [listError, setListError] = useState<Error>(
+    () => new Error('Error fetching media')
```

Comment:
This seems strange but actually kind of ok 🤔 . Would it make it more straightforward to just store the error message string in state and pass that down to the `ErrorComponent`? On the flip side, I suppose having access to the entire error might allow for a richer component.